### PR TITLE
Convert video

### PR DESCRIPTION
**Pull request by awielern**
Video converter endpoint and update of the VideoConverterModel class. Your review please

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -11,3 +11,5 @@ CONVERT_SERVICE/convert_service/convert_image/assets/*.jpg
 CONVERT_SERVICE/convert_service/convert_image/assets/*.png
 CONVERT_SERVICE/convert_service/convert_image/result/*.jpg
 CONVERT_SERVICE/convert_service/convert_image/result/*.png
+
+.idea/*
```
Comment (Franco98404) [inline .gitignore]: Please add a newline at the end of the file

File: CONVERT_SERVICE/convert_service/convert_video/model/VideoConverterModel.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#
+# @videoConverter.py Copyright (c) 2021 Jalasoft.
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/model/VideoConverterModel.py]: Please change the name of the file. Is not correct with the current file.

File: CONVERT_SERVICE/convert_service/convert_video/views.py [hunk 3]
```diff
@@ -0,0 +1,108 @@
+#
+# @videoConverter.py Copyright (c) 2021 Jalasoft.
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/views.py]: Please change the name of the file. Is not correct with the current file.

File: CONVERT_SERVICE/convert_service/convert_video/views.py [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+# Jalasoft, ("Confidential Information"). You shall not
+# disclose such Confidential Information and shall use it only in
+# accordance with the terms of the license agreement you entered into
+# with Jalasoft.
+#
+
+
+import os
+import uuid
+
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/views.py]: Please eliminate this blank line.

File: CONVERT_SERVICE/convert_service/convert_video/views.py [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+                Utilities.delete_files_pattern(pattern)
+
+                result_error = {
+                    "id": session_key,
+                    "status": "error"
+                }
+                # Result in json
+                return HttpResponse(json.dumps(result_error), 'application/json')
+
+        return HttpResponse("Please, used method POST")
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/views.py]: Please add a newline at the end of the file

File: CONVERT_SERVICE/convert_service/convert_video/admin.py [hunk 6]
```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/admin.py:14]: Please add the copyright.

File: CONVERT_SERVICE/convert_service/convert_video/apps.py [hunk 7]
```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/apps.py:14]: Please add the copyright.

File: CONVERT_SERVICE/convert_service/convert_video/models.py [hunk 8]
```diff
@@ -0,0 +1,3 @@
+from django.db import models
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/models.py:14]: Please add the copyright.

File: CONVERT_SERVICE/convert_service/convert_video/tests.py [hunk 9]
```diff
@@ -0,0 +1,3 @@
+from django.test import TestCase
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/tests.py:14]: Please add the copyright.

File: CONVERT_SERVICE/convert_service/convert_video/urls.py [hunk 10]
```diff
@@ -0,0 +1,7 @@
+from django.urls import path
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/urls.py:14]: Please add the copyright.

File: CONVERT_SERVICE/convert_service/convert_video/util/Utilities.py [hunk 11]
```diff
@@ -0,0 +1,21 @@
+import glob
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/util/Utilities.py]: Please add the copyright.

**Review by Franco98404 — COMMENTED** [review]
In the next file the copyrights are incomplete, please add the incomplete part.
CONVERT_SERVICE/convert_service/convert_service/urls.py

There are other reviews, check them. Greetings!

**Review by MatiasBonifacio — APPROVED** [review]
It okay, please solve the points that Franco mentioned before please.

File: CONVERT_SERVICE/convert_service/convert_video/model/VideoConverterModel.py (hunk 2 above)
Comment (awielern) [reply CONVERT_SERVICE/convert_service/convert_video/model/VideoConverterModel.py]: Corrected! Thanks

File: CONVERT_SERVICE/convert_service/convert_video/admin.py (hunk 6 above)
Comment (awielern) [reply CONVERT_SERVICE/convert_service/convert_video/admin.py:14]: Corrected! Thanks

File: CONVERT_SERVICE/convert_service/convert_video/apps.py (hunk 7 above)
Comment (awielern) [reply CONVERT_SERVICE/convert_service/convert_video/apps.py:14]: Corrected! Thanks

File: CONVERT_SERVICE/convert_service/convert_video/tests.py (hunk 9 above)
Comment (awielern) [reply CONVERT_SERVICE/convert_service/convert_video/tests.py:14]: Corrected! Thanks

File: CONVERT_SERVICE/convert_service/convert_video/util/Utilities.py (hunk 11 above)
Comment (awielern) [reply CONVERT_SERVICE/convert_service/convert_video/util/Utilities.py]: Corrected! Thanks

File: CONVERT_SERVICE/convert_service/convert_video/util/utilities.py [hunk 12]
```diff
@@ -0,0 +1,34 @@
+#
+# @Utilities.py Copyright (c) 2021 Jalasoft.
```
Comment (Franco98404) [inline CONVERT_SERVICE/convert_service/convert_video/util/utilities.py]: Little bit detail: Utilities.py should be with lower case

**Review by Franco98404 — APPROVED** [review]
Please check the little bit detail for the next commits. Greetings!

File: CONVERT_SERVICE/convert_service/convert_video/views.py (hunk 3 above)
Comment (awielern) [reply CONVERT_SERVICE/convert_service/convert_video/views.py]: Corrected! Thanks

File: CONVERT_SERVICE/convert_service/convert_video/views.py (hunk 5 above)
Comment (awielern) [reply CONVERT_SERVICE/convert_service/convert_video/views.py]: Space added at the end

File: CONVERT_SERVICE/convert_service/convert_video/util/utilities.py (hunk 12 above)
Comment (awielern) [reply CONVERT_SERVICE/convert_service/convert_video/util/utilities.py]: Corrected, thanks!

**Review by MatiasBonifacio — APPROVED** [review]
Everything looks good, nice job!

**Pull request merged**